pkg/commands/config/canonical_service_account.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/vmware-tanzu/kpack-cli/pkg/commands"
+	"github.com/vmware-tanzu/kpack-cli/pkg/config"
+	"github.com/vmware-tanzu/kpack-cli/pkg/k8s"
+)
+
+func NewCanonicalServiceAccountCommand(clientSetProvider k8s.ClientSetProvider) *cobra.Command {
+	var namespace string
+
+	cmd := &cobra.Command{
+		Use:          "canonical-service-account [name]",
+		Short: "Set or Get the canonical service account",
+		Long: `Set or Get the canonical service account 
+
+The canonical service account is the service account that contains the secrets for communicating with the canonical repository. It is required to be configured to use kp.
+
+This data is stored in a config map in the kpack namespace called kp-config. 
+The kp-config config map also contains the canonical repository which is teh location that imported resources and local source uploads are stored.
+
+If this config map doesn't exist, it will automatically be created by running this command, but the canonical repository field will be empty.
+`,
+		Example: `kp config canonical-service-account
```

Comment:
Perhaps when we add service account flag to `kp secret create` we can add examples to do that for the canonical-service-account?